projects/knora-ui/src/lib/viewer/values/base-value.component.ts

```diff
@@ -53,6 +41,16 @@ export abstract class BaseValueComponent {
    */
   abstract customValidators: ValidatorFn[];
 
+  standardValidatorFunc: (val: any, comment: string, commentCtrl: FormControl) => ValidatorFn = (initValue: any, initComment: string, commentFormControl: FormControl): ValidatorFn => {
```

Comment:
@mdelez I made the standard validator a member of the base class that can be overridden if necessary.